internal/controller/serving/reconcilers/kserve_raw_rolebinding_reconciler.go

```diff
@@ -0,0 +1,133 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reconcilers
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/go-logr/logr"
+	kservev1beta1 "github.com/kserve/kserve/pkg/apis/serving/v1beta1"
+	"github.com/opendatahub-io/odh-model-controller/internal/controller/constants"
+	"github.com/opendatahub-io/odh-model-controller/internal/controller/processors"
+	"github.com/opendatahub-io/odh-model-controller/internal/controller/resources"
+	v1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ SubResourceReconciler = (*KserveRawRoleBindingReconciler)(nil)
+
+type KserveRawRoleBindingReconciler struct {
+	client             client.Client
+	roleBindingHandler resources.RoleBindingHandler
+	deltaProcessor     processors.DeltaProcessor
+	serviceAccountName string
+}
+
+func NewKserveRawRoleBindingReconciler(client client.Client) *KserveRawRoleBindingReconciler {
+	return &KserveRawRoleBindingReconciler{
+		client:             client,
+		roleBindingHandler: resources.NewRoleBindingHandler(client),
+		deltaProcessor:     processors.NewDeltaProcessor(),
+		serviceAccountName: constants.KserveServiceAccountName,
+	}
+}
+
+func (r *KserveRawRoleBindingReconciler) Reconcile(ctx context.Context, log logr.Logger, isvc *kservev1beta1.InferenceService) error {
+	log.V(1).Info("Reconciling RoleBinding for InferenceService")
+	var err error
```

Comment:
can be removed.

Reply:
not sure what you want to be removed?

Reply:
The error var, if you accept the code improvement o suggested below.